client/src/components/App.jsx

```diff
@@ -46,11 +65,20 @@ class App extends React.Component {
 
   render() {
     return (
+
       <div>
```

Comment:
since you're using a `<GlobalDiv>` which will resolve to a `<div>` I don't think there is any need for the divs on line 69 and 81

Reply:
Thanks will refactor